openapi/components/paths/groups.yaml

```diff
@@ -0,0 +1,44 @@
+openapi: 3.0.3
+info:
+  title: groups
+  version: '1.0'
+  description: Groups
+paths:
+  /groups:

```

Comment:
Endpoint is not being detected because it's missing from `paths.yml`, so not included, but missing description. Single sentence what the endpoint does is enough.